docker-compose/is/conf/identity-server/repository/conf/deployment.toml

```diff
@@ -12,21 +12,28 @@ create_admin_account = true
 type = "database_unique_id"
 
 [database.identity_db]
-url = "jdbc:mysql://mysql:3306/WSO2IS_IDENTITY_DB?autoReconnect=true&amp;useSSL=false"
+type = "h2"
```

Comment:
Are we sure we have h2 as the default for 6.1.0

Reply:
According to the latest RC release, we are using h2.
$subject: Release WSO2 Identity Server 6.1.0 RC1
Isn't it @ashensw ?

Reply:
Yes H2 is the DB we use in the default pack. But it's not recommended in production. It has been the same for the previous IS version as well. But what's the expectation here? Are we recommending MySQL for docker? What did we have for IS 6?

Reply:
AFAIK, files inside docker-compose are not used in pipeline or anywhere. This file is not modified since 2019 as well. Maybe we can ignore this file

Reply:
Since it is not used, lets not worry about docker compose files that much.

Reply:
Will remove the added configs and keep it as it is then.

Reply:
Shall we not update this toml file. Anyway, the toml file is in customers' hand

Reply:
ack

Reply:
Addressed in https://github.com/wso2/docker-is/pull/366/commits/91275c755f25243bc73220b51b86c90e25515291